content/admin/org_settings/integrations/JIRA.md

```diff
@@ -48,9 +48,14 @@ Continue to the fields below to set a vulnerability status based on each JIRA ti
 
```

Comment:
As an example, here are a few mappings that can be set for Jira status / resolution combinations.